### PR TITLE
Propose a "resource list" repo within FH

### PR DESCRIPTION
**Pull request by ghost**
# Summary

Creating a good workshop takes a lot of time, a lot of love, and sometimes skills orthogonal to the content of the workshop itself.

This proposal aims to streamline the process of sharing workshops, and other resources, amongst members of the community interested in hosting workshops for their students.

# Questions & Feedback

- Do we create a repository and maintain it like an "awesome lists" repo?
- Do we simply keep and update forks of resources (esp. workshops) or do we transfer resource repositories(esp. workshops) to FloridaHackers's GitHub organization?
- Do we keep git submodules of resources (when applicable) in the repo?

### CODE REVIEW

**Comment by ghost** [comment]
Proposal todo:

- [x] check for typos and grammar
- [x] add access to FH branding to media section for purpose of making a sick README for proposed repo

**Comment by ndneighbor** [comment]
I think a good standard is making sure that when slides are submitted there are good amounts of speakers notes- and if it is a strictly technical, making sure we can get good snippets or scripts (speaking wise) for speakers.

I would totally love to have some input from actual evangelists to get some ideas on how workshop resources are shared.

@maxcell???

**Comment by maxcell** [comment]
Really really solid points (I've so far read the comments in here and will be taking a deeper dive into the shared content in a moment).

But I absolutely agree with what @Lionex says in regards of taking a lot of time and love and skill. Workshops are really hard to get right until you do even the same one a few times and iterate to see what does and doesn't work.

To address @Lionex's questions, I would suggest a repo similar to an "awesome list" namely because it allows the community to own the aspects that they build and give their identities versus just being an individual entity. This could be different if we want these workshops to strictly exist for the Florida Hackers community but it is my personal preference. In similar tokens for blog content we do at Clarifai in Medium, we have people represented in our organization but it is them as the individual contributors that recognized.

And as you said @ndneighbor, we should make sure that the supplemental content in slides exists. If there is anything that isn't directly within slides someone needs (or even if it is instructional), that content needs to exist and be available to readers. If it contains code, I would strongly suggest having a repo for your workshop and attributing that in the slides so people follow through. It also helps (in my opinion) to break things down by branch if you want it to be follow-along style. Let me take a gander of the content now and write some more thoughts. Probably intermittently within.

File: proposals/resource_list.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Workshop/Resource Repo
+
+## Target audience
+
+Organizers and community leaders within Florida hackers, or anyone else interested in hosting a workshop.
+
+## Community needs
+
+Creating a good workshop takes a lot of time, a lot of love, and sometimes skills orthogonal to the content of the workshop iteself.
```
Comment (maxcell) [inline proposals/resource_list.md]: >...content of the workshop **itself**.

File: proposals/resource_list.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Resources
+
+A GitHub repository.
+
+## Media
+
+Gwen's _Git & GitHub_ workshop shall serve as the first example of a workshop as a shared resource within Florida Hackers.
+
+Depending on the appraoch, a contributions guide shall serve to lay out precise instructions for adding a shared resource.
```
Comment (maxcell) [inline proposals/resource_list.md]: >Depending on the **approach**,...

File: proposals/resource_list.md (hunk 1 above)
Comment (ghost) [reply proposals/resource_list.md]: Yikes, thanks for spotting that one!

File: proposals/resource_list.md (hunk 2 above)
Comment (ghost) [reply proposals/resource_list.md]: And this one :+1: :joy:

**Comment by ghost** [comment]
So @maxcell, @ndneighbor, as for standards, I can think of two right away:

1. The workshop either has a permissive license or permission from the author
1. The workshop has speaker notes or supplementary content sufficient for another presenter to give the talk

Do we want to limit by topic or leave that open ended and handle on a case-by-case basis?  Any other standards that jump out to you?

Do we have an illustrative example for these standards?  I think, once the content updates are finished, my "[Git & GitHub](https://github.com/Lionex/github-workshop)" workshop comes close.

Keeping it like an "awesome lists" repo seems like the best course of action then, thanks for your input @maxcell!

**Comment by ghost** [comment]
Todo:

- [ ] final review & approval
- [ ] final style, spelling, and grammar check

**Comment by ndneighbor** [comment]
I think a case by case basis for topics works well. Like I have a technical interviewing workshop, but the slides are very conversational. That is why when I do talks I tend to have a companion blog post to pair my thoughts with the diagrams or platitudes. I agree with those base standards.

I don't have anything in addition to think about in terms of of this proposal. I guess having it be an awesome list definitely helps with having the author's equity in the talk be preserved.

Unless anyone else has any objections, this is looking like a yes from me.

(This goes back to the approval process discussion haha)

**Comment by maxcell** [comment]
HEY TIME TO :shipit:!

**Pull request merged**